lib/stream.js

```diff
@@ -63,12 +63,12 @@ function createWebSocketStream(ws, options) {
   }
 
   const duplex = new Duplex({
-    ...options,
     autoDestroy: false,
     emitClose: false,
     objectMode: false,
     readableObjectMode: false,
-    writableObjectMode: false
+    writableObjectMode: false,
+    ...options
```

Comment:
Please keep it as it originally was and just remove `objectMode`, `writableObjectMode`, `readableObjectMode` whose value is `false` by default. `autoDestroy`, and `emitClose` must never be set to `true`.

Also only `readableObjectMode` should be allowed to be changed so perhaps just remove it?

Reply:
Makes sense, done.